Assets/[SCRIPTS]/Movement.cs

```diff
@@ -1,38 +1,31 @@
-using System.Collections;
-using System.Collections.Generic;
 using UnityEngine;
 
 public class Movement : MonoBehaviour
 {
-    public float speed;
-    // Start is called before the first frame update
-    void Start()
-    {
-        speed = 10f;
-    }
+    [SerializeField] private float _speed = default;
+    [SerializeField] private float _jumpspeed = default;
+    [SerializeField] private float _movement = default;
+
+    
 
-    // Update is called once per frame
     void Update()
     {
         if (Input.GetKey(KeyCode.D))
         {
-
-            transform.Translate(1 * speed * Time.deltaTime, 0, 0);
-
+            Move();
         }
-
         if (Input.GetKey(KeyCode.A))
         {
-
-            transform.Translate(-1 * speed * Time.deltaTime, 0, 0);
-
+            transform.Translate(-1 * _speed * Time.deltaTime, 0, 0);
         }
-
         if (Input.GetKeyDown(KeyCode.Space))
         {
-
-            transform.Translate(0, 3, 0 * Time.deltaTime * speed);
-
+            transform.Translate(0, 3 * Time.deltaTime * _jumpspeed, 0);
```

Comment:
Change name of `_jumpspeed ` to `_jumpSpeed `, also `3` is not needed anymore, use the speed parameter